mcf-browser/src/Home.tsx

```diff
@@ -16,71 +16,76 @@
 
 import React, {Component} from 'react';
 
+import {Series} from './back-end/time-series';
 import {ParsingError} from './back-end/utils';
-import {openFile} from './utils';
-import {ParsingErrorsTable} from './ParsingErrorsTable';
 import {FileEntry} from './FileEntry';
 import {GraphExplorer} from './GraphExplorer';
+import {ParsingErrorsTable} from './ParsingErrorsTable';
+import {TimelineExplorer} from './TimelineExplorer';
+import {openFile} from './utils';
 
-
-interface HomePropType {
-  /**
-   * List of the files that have been uploaded by user.
-   */
-  fileList: Blob[];
-  /**
-   * Passes a file list to be submitted to the back-end for parsing.
-   */
-  upload: Function;
-  /**
-   * Passes a list of urls to be retrieved, then passed to the back-end
-   * for parsing.
-   */
-  loadFiles: Function;
-  /**
-   * Return to the home page.
-   */
-  goToHome: Function;
-  /**
-   * Clears the loaded data from all files and resets App to its initial state.
-   */
-  clear: React.MouseEventHandler<HTMLButtonElement>;
-  /**
-   * Error messages from parsing files specifying line number, line, and helpful
-   * message indicating the error.
-   */
-  errs: ParsingError[];
-  /**
-   * Indicates if uploaded files are currently being parsed.
-   */
-  loading: boolean;
-  /**
-   * IDs for nodes stored in App's state which are the subject nodes of
-   * triples from any parsed files.
-   */
-  subjNodes: string[];
-  /**
-   * Set id parameter in url to the given id. Used when user clicks a
-   * subject node to explore.
-   */
-  onNodeClick: (id: string) => void;
-}
-
-interface HomeStateType{
-  /**
-   * Determines if the file entry dropdown option should be displayed.
-   */
-  dropdown: boolean;
-}
+ interface HomePropType {
```

Comment:
this indentation looks off - I don't think there should be a space?

Reply:
fixed!